shell/src/main/java/alluxio/cli/fs/command/DuCommand.java

```diff
@@ -83,7 +98,9 @@ public String getCommandName() {
   @Override
   protected void processHeader(CommandLine cl) {
     printInfo("File Size", "In Alluxio",
-        cl.hasOption(MEMORY_OPTION_NAME) ? "In Memory" : "", "Path");
+        cl.hasOption(MEMORY_OPTION_NAME) ? Optional.of("In Memory") : Optional.empty(),
```

Comment:
What is the difference between:
`cl.hasOption(MEMORY_OPTION_NAME) ? Optional.of("In Memory") : Optional.empty()`
and 
`cl.hasOption(MEMORY_OPTION_NAME) ? "In Memory" : ""`